### PR TITLE
Abir epic 7.1 imp

### PR DESCRIPTION
**Pull request by asleem**
1- created 2 method for audio transcript
2- created 4 tests for the above methods
3- added file_type check for the file_url

### CODE REVIEW

File: .vscode/settings.json
Comment (AaronSosaRamos) [inline .vscode/settings.json:1]: This file is not needed. Please, remove it.

**Review by AaronSosaRamos — CHANGES_REQUESTED** [review]
Please, check the comment and send a screenshot of the Unit Tests that have passed appropriately.

File: .vscode/settings.json
Comment (asleem) [reply .vscode/settings.json:1]: DONE

Comment (asleem) [reply .vscode/settings.json:1]: Screenshot 2024-10-02 at 11 48 45 AM

**Review by AaronSosaRamos — APPROVED** [review]
LGTM

**Pull request merged**